2021-05-01 18:11:18,867 : INFO : Start of Prediction
2021-05-01 18:11:18,868 : INFO : run_id:2021-05-01_181118_164556733
2021-05-01 18:11:18,868 : INFO : Start of Data Load, validation and transformation
2021-05-01 18:11:18,868 : INFO : Start of Archiving Old Rejected Files...
2021-05-01 18:11:18,868 : INFO : End of Archiving Old Rejected Files...
2021-05-01 18:11:18,868 : INFO : Start of Archiving Old Validated Files...
2021-05-01 18:11:18,870 : INFO : End of Archiving Old Validated Files...
2021-05-01 18:11:18,871 : INFO : Start of Archiving Old Processed Files...
2021-05-01 18:11:18,871 : INFO : End of Archiving Old Processed Files...
2021-05-01 18:11:18,872 : INFO : Start of Archiving Old Result Files...
2021-05-01 18:11:18,872 : INFO : End of Archiving Old Result Files...
2021-05-01 18:11:18,873 : INFO : Start of Reading values From Schema...
2021-05-01 18:11:18,873 : INFO : End of Reading values From Schema...
2021-05-01 18:11:18,874 : INFO : Start of Validating Column Length...
2021-05-01 18:11:18,874 : INFO : End of Validating Column Length...
2021-05-01 18:11:18,874 : INFO : Start of Validating Missing Values...
2021-05-01 18:11:18,874 : INFO : End of Validating Missing Values...
2021-05-01 18:11:18,874 : INFO : Start of Replacing Missing Values with NULL...
2021-05-01 18:11:18,874 : INFO : End of Replacing Missing Values with NULL...
2021-05-01 18:11:18,874 : INFO : Start of Creating Table...
2021-05-01 18:11:18,875 : INFO : Opened prediction database successfully
2021-05-01 18:11:19,041 : INFO : CREATE TABLE prediction_raw_data_t column_name
2021-05-01 18:11:19,130 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:11:19,206 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:11:19,285 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:11:19,362 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:11:19,428 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:11:19,493 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:11:19,561 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:11:19,650 : INFO : ALTER TABLE prediction_raw_data_t ADD COLUMN
2021-05-01 18:11:19,650 : INFO : End of Creating Table...
2021-05-01 18:11:19,666 : INFO : Opened prediction database successfully
2021-05-01 18:11:19,666 : INFO : Start of Inserting Data into Table...
2021-05-01 18:11:19,666 : INFO : End of Inserting Data into Table...
2021-05-01 18:11:19,666 : INFO : Start of Exporting Data into CSV...
2021-05-01 18:11:19,667 : INFO : Opened prediction database successfully
2021-05-01 18:11:19,668 : INFO : End of Exporting Data into CSV...
2021-05-01 18:11:19,668 : INFO : Start of Moving Processed Files...
2021-05-01 18:11:19,669 : INFO : End of Moving Processed Files...
2021-05-01 18:11:19,670 : INFO : End of Data Load, validation and transformation
2021-05-01 18:11:19,670 : INFO : Start of Preprocessing...
2021-05-01 18:11:19,670 : INFO : Start of reading dataset...
2021-05-01 18:11:19,682 : INFO : End of reading dataset...
2021-05-01 18:11:19,682 : INFO : Start of feature encoding...
2021-05-01 18:11:19,718 : INFO : End of feature encoding...
2021-05-01 18:11:19,718 : INFO : Start of Droping Columns...
2021-05-01 18:11:19,720 : INFO : End of Droping Columns...
2021-05-01 18:11:19,720 : INFO : Start of finding missing values...
2021-05-01 18:11:19,720 : INFO : End of finding missing values...
2021-05-01 18:11:19,721 : INFO : Start of building final predictset...
2021-05-01 18:11:19,731 : INFO : End of building final predictset...
2021-05-01 18:11:19,731 : INFO : End of Preprocessing...
2021-05-01 18:11:19,731 : INFO : Start of Load Model
2021-05-01 18:11:19,732 : INFO : Model File KMeans loaded
2021-05-01 18:11:19,732 : INFO : End of Load Model
2021-05-01 18:11:19,736 : ERROR : Unsuccessful End of Prediction
Traceback (most recent call last):
  File "D:\GitHub\Employee-Retention-Prediction\App\apps\prediction\predict_model.py", line 30, in batch_predict_from_model
    clusters = kmeans.predict(self.X.drop(['empid'],axis=1))
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\cluster\_kmeans.py", line 1201, in predict
    X = self._check_test_data(X)
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\cluster\_kmeans.py", line 993, in _check_test_data
    X = check_array(X, accept_sparse='csr', dtype=[np.float64, np.float32],
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\utils\validation.py", line 72, in inner_f
    return f(**kwargs)
  File "D:\Programming\Python\Anaconda\lib\site-packages\sklearn\utils\validation.py", line 650, in check_array
    raise ValueError("Found array with %d sample(s) (shape=%s) while a"
ValueError: Found array with 0 sample(s) (shape=(0, 11)) while a minimum of 1 is required.
